src/object.d

```diff
@@ -3241,6 +3241,269 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+private template Floating1(T)
```

Comment:
Regarding the fact that this is at top level (ideally it should be nested within the only place where it's necessary): https://issues.dlang.org/show_bug.cgi?id=17242